lib/vpi/VpiImpl.cpp

```diff
@@ -168,7 +168,7 @@ GpiObjHdl* VpiImpl::create_gpi_obj_from_handle(vpiHandle new_hdl,
 GpiObjHdl* VpiImpl::native_check_create(std::string &name, GpiObjHdl *parent)
 {
     vpiHandle new_hdl;
-    std::string fq_name = parent->get_name() + "." + name;
+    std::string fq_name = parent->get_fullname() + "." + name;
```

Comment:
Isn't `get_fullname` going to be different depending on the impl?

For example VHDL often as `parent:child:object` whereas we're effectively enforcing a `parent.child.object` style by inserting the `.` ourselves.